src/pybind/mgr/zabbix/module.py

```diff
@@ -269,30 +288,32 @@ def send(self):
             })
             return
 
-        try:
+        result = True
+
+        for server in self._zabbix_hosts:
             self.log.info(
-                'Sending data to Zabbix server %s as host/identifier %s',
-                self.config['zabbix_host'], identifier)
+                'Sending data to Zabbix server %s, port %s as host/identifier %s',
+                server['zabbix_host'], server['zabbix_port'], identifier)
             self.log.debug(data)
 
-            zabbix = ZabbixSender(self.config['zabbix_sender'],
-                                  self.config['zabbix_host'],
-                                  self.config['zabbix_port'], self.log)
-
-            zabbix.send(identifier, data)
-            self.set_health_checks(dict())
-            return True
-        except Exception as exc:
-            self.log.error('Exception when sending: %s', exc)
-            self.set_health_checks({
-                'MGR_ZABBIX_SEND_FAILED': {
-                    'severity': 'warning',
-                    'summary': 'Failed to send data to Zabbix',
-                    'detail': [str(exc)]
-                }
-            })
-
-        return False
+            try:
+                zabbix = ZabbixSender(self.config['zabbix_sender'],
+                                      server['zabbix_host'],
+                                      server['zabbix_port'], self.log)
+                zabbix.send(identifier, data)
+            except Exception as exc:
+                self.log.error('Exception when sending: %s', exc)
```

Comment:
if you're using `self.log.exception("failed to send.")`, it will also automatically print the traceback

Reply:
Changed in last commit.